ecosystem/platform/server/app/components/header_component.html.erb

```diff
@@ -31,7 +31,7 @@
 
   <% if @user %>
     <div class="relative">
-      <button data-action="click->header#toggleUser" class="w-8 h-8 bg-transparent border border-1 border-teal-400 ring-teal-400 hover:ring-1 hover:bg-neutral-800 text-teal-300 font-semibold rounded-lg inline-flex items-center justify-center overflow-hidden">
+      <button data-action="click->header#toggleUser" class="w-8 h-8 bg-transparent ring-1 ring-teal-400 hover:ring-2 hover:bg-teal-400 hover:text-neutral-900 hover:brightness-105 active:brightness-95 text-teal-300 font-semibold rounded-lg inline-flex items-center justify-center overflow-hidden">
```

Comment:
Does it / should it have a ring on focus?

Reply:
Will put some focus into this soon as we further refine additional buttons.